src/bfcli/opts.c

```diff
@@ -187,6 +189,14 @@ static const struct bfc_opts_cmd _bfc_opts_cmds[] = {
     },
 };
 
+void bf_print_version(FILE *stream, struct argp_state *state)
+{
+    UNUSED(state);
+
+    (void)fprintf(stream, "bfcli version %s, libbpfilter version %s\n",
+                  BF_VERSION, bf_version());
+}
+
```

Comment:
Move this back to `bfcli/main.c` as `static void _bfc_print_version...)`: it's not related to the options per se, only a helper function.